mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -76,11 +111,18 @@ export const AddOrRemoveCollateralForm = React.memo(({ route }: Props): JSX.Elem
     validateInput(collateralValue)
   }, [collateralValue])
 
+  const bottomSheetComponents = {
+    mobile: BottomSheetScrollView,
+    web: ThemedScrollView
+  }
+  const ScrollView = Platform.OS === 'web' ? bottomSheetComponents.web : bottomSheetComponents.mobile
+
   return (
-    <ThemedScrollView
-      light={tailwind('bg-white')}
-      dark={tailwind('bg-gray-800')}
-      style={tailwind('p-4 flex-1')}
+    <ScrollView
```

Comment:
curious on why not `<ThemedScrollView>` here?

Reply:
Yea the scrolling not working on android properly with bottomsheet so had to use: `BottomSheetScrollView` on mobile
https://github.com/DeFiCh/wallet/pull/1740/files#diff-4d1e25797b150b1e0b215298574cd18daf99ff2c3264dadb644fda765e905d6eR114